.github/workflows/run-mayavi-tests.yml

```diff
@@ -1,4 +1,7 @@
 name: Tests
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number }}-${{ github.event.type }}
```

Comment:
can I ask why you chose this specifically?

Reply:
The three categories separate workflows, PRs, and push-vs-PR from each other. If you just had for example `${{ github.workflow }}`, pushing a commit to one PR would kill the runs in all others and and `main` builds (though you don't have any `main` builds currently)

Reply:
thank you very much! merging